.github/dependabot.yaml

```diff
@@ -37,7 +37,8 @@ updates:
   # Update our Dockerfile.
   - package-ecosystem: "docker"
     directories:
-      - "/dogfood/contents"
+      - "/dogfood/coder/contents"
+      - "/dogfood/coder-envbuilder/contents"
```

Comment:
I don't see a big use of the `contents` sub directly, as we now have individual directories for each template.

Reply:
Done!